src/main/resources/static/static/js/updateOrder.js

```diff
@@ -0,0 +1,68 @@
+//计算商品总价
+function updateFinalPrice() {
+    var $tr = $("#itemTable").find("tr[id]");
+    var summer = 0;
+    $tr.each(function(i, dom) {
+        if($(dom).find("#sItemChecked").is(":checked")){
+            var num = $(dom).find(".ItemNo").val(); //商品数量
+            var name = $(dom).find("#ItemName").text();//商品名称
+            if(num < 1){
+                alert("请修改商品:"+ name+" 数量！");
+            }else{
+                var price = num * $(dom).find(".sPrice").text(); //商品小计
+                summer += price; //总价
+            }
+        }
+    });
+    $("#finalPrice").text(summer);
+}
+//提交订单
+function submitOrder() {
+
+}
+//更新购物车
+function updateCart() {
+    var $tr = $("#itemTable").find("tr[id]");
+    $tr.each(function(i, dom) {
```

Comment:
Same problem, do not name a variable like this.

Reply:
solved

Reply:
Commit and push your changes.